clients/admin-ui/src/features/datastore-connections/ConnectionFilters.tsx

```diff
@@ -0,0 +1,179 @@
+import {
+  Input,
+  InputGroup,
+  InputLeftElement,
+  Select,
+  Stack,
+  useToast,
+} from "@fidesui/react";
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import { selectToken } from "../auth";
+import { SearchLineIcon } from "../common/Icon";
+import { capitalize } from "../common/utils";
+import {
+  clearAllFilters,
+  requestCSVDownload,
+  selectPrivacyRequestFilters,
+  setRequestFrom,
+  setRequestId,
+  setRequestStatus,
+  setRequestTo,
+} from "../privacy-requests";
+import { PrivacyRequestStatus } from "../privacy-requests/types";
+import { ConnectionType, SystemType, TestingStatus } from "./types";
+
+const useConstantFilters = () => {
+  const filters = useSelector(selectPrivacyRequestFilters);
+  const token = useSelector(selectToken);
+  const dispatch = useDispatch();
+  const toast = useToast();
+  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) =>
+    dispatch(setRequestId(event.target.value));
+  const handleStatusChange = (event: React.ChangeEvent<HTMLSelectElement>) =>
```

Comment:
Looks like something in this function is causing the other selects to deselect whenever another element is selected

Reply:
Right now the dropdowns are more of a placeholder. The ones in this PR are replaced and fixed within https://github.com/ethyca/fidesops/pull/691